### PR TITLE
Make the module runnable as a script

### PR DESCRIPTION
**Pull request by ao-picterra**
This is smth expected to be internal, and may turn useful for workflows

### CODE REVIEW

File: src/picterra/__main__.py [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+    list_parser = subparsers.add_parser('list', help="List resources")
+    list_parser.add_argument(
+        'resource', help="List either detectors or rasters in your account",
+        type=str, choices=['rasters', 'detectors'])
+
+    # create the parser for the "detect" command
+    detect_parser = subparsers.add_parser('detect', help="Predict on a raster with a detector")
+    detect_parser.add_argument("raster", help="ID of a raster", type=str)
+    detect_parser.add_argument("detector", help="ID of a detector", type=str)
+    detect_parser.add_argument("path", help="Path of the file were results will be saved", type=str)
```
Comment (restaste) [inline src/picterra/__main__.py:34]: What about something a bit more explicit, like `output_file` ?

File: src/picterra/__main__.py [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+    create_annotation_parser.add_argument("raster", help="ID of a raster", type=str)
+    create_annotation_parser.add_argument("detector", help="ID of a detector", type=str)
+    create_annotation_parser.add_argument(
+        "type", help="Type of the annotation", type=str,
+        choices=['outline', 'training_area', 'testing_area', 'validation_area'])
+    # create detection area
+    create_detection_area_parser = create_subparsers.add_parser(
+        'detection_area', help="Add a detection area to a raster")
+    create_detection_area_parser.add_argument(
+        "path", help="Path where retrieving the detection area geometries",
```
Comment (restaste) [inline src/picterra/__main__.py:85]: nit: what about "Path to the geojson file containing the detection area geometries"?

File: src/picterra/__main__.py [hunk 3]
```diff
@@ -0,0 +1,189 @@
...
+        "--training-steps", metavar='',
+        help="Number of steps while training the detector: an integer in the range 500..40000",
+        type=int, default=500)
+    create_detector_parser.add_argument(
+        "-r", "--raster", help="ID(s) of the raster (s) to associate with the detector",
+        type=str, required=False, nargs='+', default=[])
+    # create raster
+    create_raster_parser = create_subparsers.add_parser(
+        'raster', help="Creates a raster, optionally adding it to a detector")
+    create_raster_parser.add_argument("path", help="Path where retrieving the raster", type=str)
```
Comment (restaste) [inline src/picterra/__main__.py:63]: nit: what about "Path to the raster file"?

**Review by restaste — COMMENTED** [review]
Looks great! Couple of nitpicks, but LGTM!

**Comment by julienr** [comment]
Cool idea ! :)

**Review by restaste — APPROVED** [review]

**Pull request merged**